learning_resources/etl/openedx.py

```diff
@@ -233,6 +241,50 @@ def _transform_image(image_data: dict) -> dict:
     return None
 
 
+def _transform_program_image(program_data) -> dict:
+    """Return the transformed image data if a url is provided"""
+    url = program_data.get("banner_image", {}).get("medium", {}).get("url")
+    if url:
+        return {"url": url, "description": program_data.get("title")}
+    return None
+
+
+def _parse_course_dates(program, date_field):
+    """Return all the course run price values for the given date field"""
+    dates = []
+    for course in program.get("courses", []):
+        if not course["excluded_from_search"]:
+            dates.extend(
+                [
+                    run[date_field]
+                    for run in course["course_runs"]
+                    if run["status"] == "published" and run[date_field]
+                ]
+            )
+    return dates
+
+
+def _add_course_prices(program):
```

Comment:
I'd suggest changing the name of this function since `_add_course_prices` sounds like it'd add the course prices to the program object being passed in. Replacing "add" with "sum" is probably good enough.